.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing Guidelines
+
+-   You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (for example #powerup or #portal). Bridge in Tech stream is _blank_.
```

Comment:
Pls make the necessary changes - 
1. Bridge in Tech should be Open Source Programs instead.
2. Add "Open Source Programs stream is [#open-source-progs](https://anitab-org.zulipchat.com/#streams/237907/open-source-progs)." instead.

Thank you, do make necessary changes in the issue  - add contributing.md